meerk40t/core/svg_io.py

```diff
@@ -646,40 +670,53 @@ def parse(self, element, context_node, e_list):
         else:
             # Check if SVGElement:  Note.
             tag = element.values.get(SVG_ATTR_TAG)
+            # We need to reverse the meerk40t:... replacement that the routine had already applied:
             if tag is not None:
                 tag = tag.lower()
+                torepl = "{" + MEERK40T_NAMESPACE.lower() + "}"
+                if torepl in tag:
+                    oldval = element.values.get(tag)
+                    replacer = MEERK40T_XMLS_ID.lower() + ":"
+                    tag = tag.replace(torepl, replacer, )
+                    element.values[tag] = oldval
+                    element.values[SVG_ATTR_TAG] = tag
             if tag == "note":
                 self.elements.note = element.values.get(SVG_TAG_TEXT)
                 self.elements.signal("note", self.pathname)
                 return
             node_type = element.values.get("type")
             if node_type is None or node_type == "op":
                 # Meerk40t 0.7.x fallback node types.
-                op_type = element.values.get("operation")
+                op_type = element.values.get(MEERK40T_XMLS_ID + ":operation")
                 if op_type is None:
-                    return
+                    op_type = element.values.get("operation")
+                    if op_type is None:
+                        return
                 node_type = f"op {op_type.lower()}"
 
             if node_type is not None:
                 node_id = element.values.get("id")
                 # Check if SVGElement: operation
-                if tag == "operation":
+                op_s1 = MEERK40T_XMLS_ID + ":operation"
+                op_s2 = "operation"
+                if tag == op_s1.lower() or tag == op_s2.lower():
```

Comment:
op_s1 and op_s2 are defined by us and don't need the lower()
`if tag.endswith("operation")`? `if tag is in ("operation", MEER40T_XMLS_ID:operation)`...

Reply:
Done